content/documentation/runtimes-monitoring/index.adoc

```diff
@@ -268,7 +281,10 @@ Spec fields definitions are:
 *** *chartType*: type of the chart, can be one of _line_ (default), _area_ or _bar_
 *** *unit*: unit for Y-axis. Free-text field to provide any unit suffix. It can eventually be scaled on display. See link:#units[specific section below].
 *** *spans*: number of "spans" taken by the chart, from 1 to 12, using link:https://www.w3schools.com/bootstrap4/bootstrap_grid_system.asp[bootstrap convention]
-*** *metricName*: the metric name in Prometheus
+*** _metricName (deprecated starting from Kiali 1.18)_: the metric name in Prometheus; this field is deprecated, use *metrics* instead.
+*** *metrics* (since Kiali 1.18): the list of metrics to display:
```

Comment:
we are close to having versioned docs, and then we should be able to stop making inline comments like this. fine for now.